web/components/FcPaneMap.vue

```diff
@@ -101,11 +101,24 @@ const PAINT_WIDTH_MIDBLOCKS = interactionAttr(3, 5, 5);
 const PAINT_RADIUS_INTERSECTIONS = interactionAttr(8, 10, 10);
 const PAINT_RADIUS_COUNTS = interactionAttr(10, 12, 12);
 
+function getTileOrigin() {
```

Comment:
i'd prefer this to use `https://flashcrow-etladmin.intra.dev-toronto.ca` in all environments for now - we just need to add the `/glyphs` path in nginx configs on `etl`.